docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -96,6 +96,108 @@ def do_stuff_partitioned():
 
 In addition to the <PyObject object="daily_partitioned_config" decorator /> decorator, Dagster also provides <PyObject object="monthly_partitioned_config" decorator />, <PyObject object="weekly_partitioned_config" decorator />, <PyObject object="hourly_partitioned_config" decorator />.
 
+By default, these decorators create partitions as follows:
+
+<PyObject object="hourly_partitioned_config" decorator /> - a partition is a single
+hour spanning minute 0 to minute 0. For example, defining a partition:
+
+```python file=/concepts/partitions_schedules_sensors/partition_defs.py startafter=start_hourly_partition_def endbefore=end_hourly_partition_def
+@hourly_partitioned_config(start_date=datetime(2022, 3, 12))
+def my_hourly_partitioned_config(start: datetime, _end: datetime):
+    return {
+        "ops": {
+            "process_data_for_date": {"config": {"date": start.strftime("%Y-%m-%d")}}
+        }
+    }
+```
+
+will create partitions of the form `(2022-03-12T00:00, 2022-03-12T01:00), (2022-03-12T01:00, 2022-03-12T02:00), ...`
```

Comment:
The format we display with in dagit looks like `2022-03-12-00:00` - so possibly worth using that here? Also, I think it's worth mentioning how the partitions will be displayed on the dagit partitions page (i.e. that we use the start of the window). https://demo.elementl.show/workspace/hacker_news_assets_prod@hacker-news-assets/jobs/hacker_news_api_download/partitions